pig/src/main/java/org/jboss/pnc/bacon/pig/impl/config/RepoGenerationStrategy.java

```diff
@@ -22,6 +22,8 @@
  *         Date: 5/23/18
  */
 public enum RepoGenerationStrategy {
-    GENERATE, DOWNLOAD, BUILD_CONFIGS, BUILD_GROUP, MILESTONE, OFFLINER_ONLY, IGNORE, PACK_ALL // DEPRECATED replaced by
-                                                                                               // BUILD_CONFIGS
+    RESOLVE_ONLY, GENERATE, DOWNLOAD, BUILD_CONFIGS, BUILD_GROUP, MILESTONE, OFFLINER_ONLY, IGNORE, PACK_ALL // DEPRECATED
```

Comment:
maybe `RESOLVE_BOM_DEPENDENCIES` instead of `RESOLVE_ONLY`?

Reply:
Sure, that makes more sense